### ISSUE TITLE
Plover doesn't work with the on-screen windows keyboard

### ISSUE THREAD

**Issue by moyamo** [openstenoproject/plover#873]
## Summary

Plover doesn't work with the on-screen windows keyboar.

## Reproducing

Enable plover, type with the onscreen keyboard. The normal QWERTY keys are output instead of the plover words.

Using a USB keyboard works.

## Plover Version

3.1.1

## System

Windows 10

**Comment by morinted** [comment]
I'm pretty sure the Windows on screen keyboard doesn't support "holding" keys, which would be required for Plover to work with it.

**Comment by morinted** [comment]
A better option might be a dedicated machine plugin with a QT tool that displays an on-screen-steno-machine from within Plover. That should work with any 10-point multitouch display.

**Comment by moyamo** [comment]
I agree that a on screen steno keyboard will probably be the best solution. If it's within the scope of this project I wouldn't mind helping (although I know nothing about Windows Development or stenography).

**Comment by morinted** [comment]
I think it's a good idea for a plugin, but it's not something I'd personally like to take on. We can leave this issue open and perhaps we'll find someone who's up to take it.

**Comment by stillgreyfox** [comment]
FWIW, just tried Win 10 onscreen keyboard with multitouch laptop screen.
It does let you hold down multiple keys, so you can do CTRL-A, CTRL-C, etc.

Tried it with web demo, and it only registers one key on release, so, av turns into a, then v.

I'm introducing a local Python meetup to Plover this week, and will see if they're interested in taking on this issue.

**Comment by Zireael07** [comment]
Bump! Any news?

**Comment by user202729** [comment]
@Zireael07 Searching around there's this plugin https://github.com/wooningeire/plover-touchscreen-stenotype , although without a touch screen I cannot really test it.

Consider this issue fixed then?

**Comment by mkrnr** [comment]
Closing this one due to its age and since it's a plugin candidate.
